plugins/node/instrumentation-mongoose/test/mongoose.test.ts

```diff
@@ -63,8 +63,11 @@ describe('mongoose instrumentation', () => {
   beforeEach(async () => {
     instrumentation.disable();
     instrumentation.setConfig({
-      dbStatementSerializer: (_operation: string, payload) =>
-        JSON.stringify(payload),
+      dbStatementSerializer: (_operation: string, payload) => {
+        return JSON.stringify(payload, (key, value) => {
+          return key === 'session' ? '[Session]' : value;
```

Comment:
ℹ️ Note this replacer function was only introduced here to avoid an error due to the Session object having cyclical reference